src/modules/utils/motordrivercontrol/MotorDriverControl.h

```diff
@@ -35,14 +36,15 @@ class MotorDriverControl : public Module {
         void dump_status(StreamOutput*, bool);
         void set_raw_register(StreamOutput *stream, uint32_t reg, uint32_t val);
         void set_options(Gcode *gcode);
-
         void enable(bool on);
-        int sendSPI(uint8_t *b, int cnt, uint8_t *r);
+		void WriteReadSPIstr(const char *p, StreamOutput *stream);
```

Comment:
Tabs are not what we use, please make sure your whitespace conforms to our coding standards, 4 spaces per indent. Thank you.

Reply:
Replaced tabs with spaces.